### ISSUE TITLE
Vagrant Destroy Fails

### ISSUE THREAD

**Issue by matthew-a-dunlap** [IQSS/dataverse#4519]
When using vagrant, if you attempt to destroy a created box it throws this error:
```
There are errors in the configuration of this machine. Please fix
the following errors and try again:

vm:
* A box must be specified.
```
There is a workaround in our Vagrantfile (look for this line: _# Uncomment this temporarily to get \`vagrant destroy\` to work_). But using this workaround causes other issues with vagrant. 

This was not a problem previously and we should find a fix to allow vagrant use to be smooth.

**Comment by pdurbin** [comment]
@matthew-a-dunlap thanks for reporting this. It's something @donsizemore and I talked about in #4384 but never resolved.

The fix might be to switch to a different base box. I'm not sure.

**Comment by matthew-a-dunlap** [comment]
@pdurbin No problem. For what its worth, I uncommented the "fix" line and saw no issues in getting a vagrant box up.

**Comment by pdurbin** [comment]
I fixed this in a7c986d. Closing.
